src/client/editor/Editor.js

```diff
@@ -147,65 +159,42 @@ export default class Editor {
 
   //
 
-  _updateBreadCrumb(breadCrumb) {
-    const uriParts = this.sceneURI.split("/");
-    breadCrumb.name = uriParts[uriParts.length - 1];
-    breadCrumb.uri = this.sceneURI;
+  popScene() {
+    this.scenes.pop();
+    const { uri, obj } = last(this.scenes);
+    this.setSceneURI(uri);
+    this.setScene(obj);
   }
 
   setSceneURI(uri) {
-    this.sceneURI = uri;
-    if (this.breadCrumbs.length === 0) {
-      this.breadCrumbs.push({});
-    }
-    this._updateBreadCrumb(this.breadCrumbs[this.breadCrumbs.length - 1]);
+    const sceneInfo = last(this.scenes);
+    sceneInfo.uri = uri;
   }
 
-  _loadScene(url) {
+  _loadAndSetScene(url) {
     return loadScene(url, this.gltfComponents, true).then(scene => {
-      this.breadCrumbs.push({});
-      this.setSceneURI(url);
+      this.scenes.push({ uri: url, obj: scene });
       this.setScene(scene);
       return scene;
     });
   }
 
-  openScene(url) {
-    this.breadCrumbs = [];
-    return this._loadScene(url);
+  editScenePrefab(url) {
+    this._loadAndSetScene(url);
   }
 
-  setScene(scene) {
-    this.scene.uuid = scene.uuid;
-    this.scene.name = scene.name;
-
-    if (scene.background !== null) this.scene.background = scene.background.clone();
-    if (scene.fog !== null) this.scene.fog = scene.fog.clone();
-
-    this.scene.userData = JSON.parse(JSON.stringify(scene.userData));
-
-    // avoid render per object
-
-    this.signals.sceneGraphChanged.active = false;
-
-    while (this.scene.children.length > 0) {
-      this.removeObject(this.scene.children[0]);
-    }
-
-    while (scene.children.length > 0) {
-      this.addObject(scene.children[0]);
-    }
-
-    this.signals.sceneGraphChanged.active = true;
```

Comment:
Not sure if there was any point to replacing children in the scene instead of just using the new scene itself.